src/features/user/usersSlice.tsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-param-reassign */
```

Comment:
It's better to avoid using 'no-param-reassign' rule disabling, try to use existing state manipulation functions from Redux Toolkit